packages/core/src/cart/buy-now-cart-request-body.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * An object that contains the information required for creating 'Buy now' cart.
+ */
+export default interface BuyNowCartRequestBody {
+    source: 'BUY_NOW';
+    lineItems: Array<{
```

Comment:
Can you use the existing lineItems interface?

Reply:
No, because we have different interfaces for line items data what we pass for cart creation, and the line items data what we get from BE after cart creation

<img width="631" alt="Screenshot 2022-09-21 at 16 23 54" src="https://user-images.githubusercontent.com/25133454/191516102-c9722f7a-4cda-4062-ae4e-1942d83fb64d.png">